rbac/management/permission/model.py

```diff
@@ -26,6 +26,7 @@ class Permission(models.Model):
     resource_type = models.TextField(null=False)
     verb = models.TextField(null=False)
     permission = models.TextField(null=False, unique=True)
+    description = models.TextField(default="")
```

Comment:
Hey @astrozzc. for `Permission` objects already created, the serializer change to add the `description` field will surface something like the following:
```json
{
  "application": "ansible-automation",
  "resource_type": "*",
  "verb": "*",
  "permission": "ansible-automation:*:*",
  "description": null
}
```

However for a newly created tenant, I'll see the following when there's no `description`, because it's being created as a blank string?
```json
{
  "application": "approval_local_test",
  "resource_type": "workflows",
  "verb": "update",
  "permission": "approval_local_test:workflows:update"
}
```
Which omits the field altogether when it's `""`, vs `null` or has a value.

Reply:
Not sure if maybe for consistency we should update the seed logic to set it to `None` by default, and have `default=None` in the model? And just don't use the `to_representation` so the payload always has the same fields?

Reply:
Hmm, we have already run migrations for the db to set all the created permission with that field as empty string "".
I am wondering where those null come from. 
How do you reproduce the issue? I tried reinit database, create the tenant, check and then apply the change, but do not have this error.

Reply:
I'm seeing the same behavior here, reproduction steps for me were:
1. `git checkout master`
2. `make reinitdb`
3. `make serve`
4. `curl localhost/api/rbac/v1/permissions/` (to create tenant)
5. `curl localhost:8000/api/rbac/v1/roles/ -X POST -H 'content-Type: application/json' -d '{"name": "testy", "access": [{"permission": "cost-management:*:*", "resourceDefinitions": [{"attributeFilter":{"key":"sa"
,"value":"sda","operation":"in"}}}]}]}'`  (to modify new tenant)
6. check out PR branch
7. `make serve` (if not still running)
8 ` curl localhost:8000/api/rbac/v1/permissions/`

Output for me contained null descriptions:

`{
  "meta": {
    "count": 66,
    "limit": 10,
    "offset": 0
  },
  "links": {
    "first": "/api/rbac/v1/permissions/?limit=10&offset=0",
    "next": "/api/rbac/v1/permissions/?limit=10&offset=10",
    "previous": null,
    "last": "/api/rbac/v1/permissions/?limit=10&offset=56"
  },
  "data": [
    {
      "application": "ansible-automation",
      "resource_type": "*",
      "verb": "*",
      "permission": "ansible-automation:*:*",
      "description": null
    },
    {
      "application": "approval",
      "resource_type": "workflows",
      "verb": "create",
      "permission": "approval:workflows:create",
      "description": null
    },
--->8---
`